src/tools/pub/package-layout.md

```diff
@@ -115,12 +114,12 @@ enchilada/
 
 One file that's very common in open source is a README file that
 describes the project. This is especially important in pub. When you upload
-to the [Pub site]({{site.pub}}), your README is shown on
+to the [Pub site,]({{site.pub}}) your README is shown on
 the page for your package. This is the perfect place to introduce people to
 your code.
 
-If your README ends in `.md`, `.markdown`, or `.mdown`, it is parsed as
-[Markdown][].
+If your README ends in `.md`, `.markdown`, or `.mdown`, it's parsed as
```

Comment:
Can we remove `.markdown` and `.mdown`.

I think `.md` has won on GitHub, and there is no reason for us to suggest that users use anything other than `README.md`. Even if we also support other filenames at the moment.